tests/test_models/standard_model_tests.py

```diff
@@ -11,18 +11,34 @@
 class StandardModelTest(object):
     """ Basic processing test for the models. """
 
-    def __init__(self, model):
+    def __init__(
+        self,
+        model,
+        parameter_values=None,
+        geometry=None,
+        submesh_types=None,
+        var_pts=None,
+        spatial_methods=None,
+        solver=None,
+    ):
         self.model = model
-        # Set default parameters
-        self.parameter_values = model.default_parameter_values
+        # Set parameters, geometry, spatial methods etc
+        # The code below is equivalent to:
+        #    if parameter_values is None:
+        #       self.parameter_values = model.default_parameter_values
+        #    else:
+        #       self.parameter_values = parameter_values
+        self.parameter_values = parameter_values or model.default_parameter_values
+        geometry = geometry or model.default_geometry
+        self.submesh_types = submesh_types or model.default_submesh_types
```

Comment:
Maybe does not really matter but submesh_types, var_pts, and spatial_methods don't need to be properties of the class.